onyo/commands/new.py

```diff
@@ -68,8 +68,14 @@
 def new(args: argparse.Namespace) -> None:
     """
     Create new ``DIRECTORY/ASSET``\\(s), and add contents with ``--template``,
-    ``--keys`` and ``--edit``. If the directories do not exist, they will be
-    created.
+    ``--keys``, ``--clone`` and ``--edit``. If the directories do not exist,
+    they will be created.
+
+    When the same key is specified multiple times with ``onyo new --keys [...]``
+    onyo creates a new asset for each of them. A key that is specified just
+    one time will be applied to all new assets.
+    All keys that are used more than one time must be used the same time to
```

Comment:
"All keys that are used more than one time must be used the same time"

This doesn't make sense to me. I think what you're trying to describe is that if creating two assets, and setting keys for them, all keys need to be specified two times. I think an example will also help here.

Reply:
That is not exactly what I try to describe, which does underlines the need for adjustment:

"When creating X new assets, all keys must be specified exactly 1 or X to times; if specified once, a key will applied to all assets with the same value, otherwise they will be applied to each new asset in the order of specification."

Reply:
I like this much better. Way more clear.

One oops:

> 1 or X to time

Remove "to".

Reply:
Great catch 🤦